packages/frontend/package.json

```diff
@@ -51,8 +51,7 @@
     "tweetnacl": "^1.0.3",
     "tweetnacl-util": "^0.15.1",
     "url-parse": "^1.4.7",
-    "uuid": "^8.0.0",
-    "xstate": "^4.9.1"
```

Comment:
@LevelbossMike should ember-statecharts have xstate as a peer dependency? 
I've never known what the best sub-dep strategy is.

 - move it to the app's responsibility so that the app can be more up to date if they want
 - bundle it in the addon so that the app doesn't need to worry about

Reply:
I thought about this as well. Ideally I had the idea in mind that it should be the responsibility of the host app to install it. 

With `ember-statecharts` right now though it's bundled because we make use of `ember-auto-import` to be able to use xstate directly. If you have addon code that needs to be able to do `import xstate from 'xstate'` you need to declare it as a dependency of the addon, i.e. bundle it up for usage.

If you want to make use of another xstate version you can declare it as a dependency of the host app and make use of resolutions to tell yarn (I guess that works for npm too) what version you want installed.

Problem with both approaches (responsibility of host app to install, or use resolutions to customize) is the same. You can't be sure that the xstate version you are using works with `ember-statecharts`. Right now if you use `ember-statecharts` out of the box and use the xstate version it bundles you can be sure that it works - as this is tested in `ember-statecharts`' tests.